spring-cloud-gcp-data-spanner/src/test/java/org/springframework/cloud/gcp/data/spanner/core/SpannerTemplateTests.java

```diff
@@ -641,6 +641,33 @@ public void resolveChildEntityTest() {
 		});
 	}
 
+	@Test
+	public void lazyFetchChildrenTest() {
+		ChildEntity c = new ChildEntity();
```

Comment:
Any reason these ChildrenEntity's fields are not set through constructor?

Reply:
yes :D such a constructor does not exist